provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "aws",
+    "displayName": "AWS",
```

Comment:
I assume aws-native is gonna be called `AWS Cloud Control`?

Reply:
That's what it's called now:

https://github.com/pulumi/registry/blob/eb8278beb938bedbac4f28c911a1cd50644fa2fb/tools/resourcedocsgen/pkg/lookup.go#L131